BLUESPAWN-client/headers/mitigation/mitigations/MitigateM1025.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include "mitigation/Mitigation.h"
+#include "mitigation/MitigationRegister.h"
+#include "hunt/reaction/Reaction.h"
+#include "hunt/reaction/Log.h"
+
+namespace Mitigations{
+
+	/**
+	 * MitigateM1042-LLMNR looks for LSA to be run as a protected process light,
```

Comment:
need to fix comment here then this will be good

Reply:
I will do this later today

Reply:
Done